Description: Install missing files
 PBR is missing the templates folder.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2017-11-07

--- /dev/null	2017-10-16 21:43:57.398818308 +0000
+++ b/MANIFEST.in	2017-11-07 19:39:31.124537693 +0000
@@ -0,0 +1,2 @@
+recursive-include trove/locale *
+recursive-include trove/templates *
